Documentation/MsTest.md

```diff
@@ -2,4 +2,212 @@
+dotnet_diagnostic.MSTEST0007.severity = error
+dotnet_diagnostic.MSTEST0008.severity = error
+dotnet_diagnostic.MSTEST0009.severity = error
+dotnet_diagnostic.MSTEST0010.severity = error
+dotnet_diagnostic.MSTEST0011.severity = error
+dotnet_diagnostic.MSTEST0012.severity = error
+dotnet_diagnostic.MSTEST0013.severity = error
+dotnet_diagnostic.MSTEST0014.severity = error
+dotnet_diagnostic.MSTEST0015.severity = error
+dotnet_diagnostic.MSTEST0016.severity = error
+dotnet_diagnostic.MSTEST0017.severity = error
+dotnet_diagnostic.MSTEST0018.severity = error
+dotnet_diagnostic.MSTEST0019.severity = warning  # Exception: not set to error
+dotnet_diagnostic.MSTEST0020.severity = error
+dotnet_diagnostic.MSTEST0021.severity = error
+dotnet_diagnostic.MSTEST0022.severity = error
+dotnet_diagnostic.MSTEST0023.severity = error
+dotnet_diagnostic.MSTEST0024.severity = error
+dotnet_diagnostic.MSTEST0025.severity = error
+dotnet_diagnostic.MSTEST0026.severity = error
+dotnet_diagnostic.MSTEST0027.severity = error
+dotnet_diagnostic.MSTEST0028.severity = error
+dotnet_diagnostic.MSTEST0029.severity = error
+dotnet_diagnostic.MSTEST0030.severity = error
+dotnet_diagnostic.MSTEST0031.severity = error
+dotnet_diagnostic.MSTEST0032.severity = error
+dotnet_diagnostic.MSTEST0033.severity = error
+dotnet_diagnostic.MSTEST0034.severity = error
+dotnet_diagnostic.MSTEST0035.severity = error
+dotnet_diagnostic.MSTEST0036.severity = error
+dotnet_diagnostic.MSTEST0037.severity = error
+dotnet_diagnostic.MSTEST0038.severity = error
+dotnet_diagnostic.MSTEST0039.severity = error
+dotnet_diagnostic.MSTEST0040.severity = error
+dotnet_diagnostic.MSTEST0041.severity = error
+dotnet_diagnostic.MSTEST0042.severity = error
+dotnet_diagnostic.MSTEST0043.severity = error
+dotnet_diagnostic.MSTEST0044.severity = error
+dotnet_diagnostic.MSTEST0045.severity = error
+dotnet_diagnostic.MSTEST0046.severity = error
+dotnet_diagnostic.MSTEST0047.severity = error
+dotnet_diagnostic.MSTEST0048.severity = error
+dotnet_diagnostic.MSTEST0049.severity = error
+dotnet_diagnostic.MSTEST0050.severity = error
+
+# Keep Philips-specific rules as needed
+dotnet_diagnostic.PH2000.severity = suggestion  # Test method naming
+dotnet_diagnostic.PH2012.severity = warning     # Test timeout required
+dotnet_diagnostic.PH2015.severity = warning     # Required categories
+```
+
+### 3. Consider Removing Philips MSTest Package
+
+Once migration is complete and you've verified the Microsoft analyzers work for your needs, consider removing the Philips package if you don't find sufficient value from the remaining rules:
+
+```xml
+<!-- Remove this when ready -->
+<PackageReference Include="Philips.CodeAnalysis.MsTestAnalyzers" Version="1.0.0" PrivateAssets="all" />
+```
+
+## Benefits of Migration
+
+1. **Official Support**: Backed by the MSTest team at Microsoft
+2. **Better Integration**: Designed specifically for MSTest framework
+3. **More Comprehensive**: 46+ rules vs 30 Philips rules
```

Comment:
Seems a repetition of earlier statements

Reply:
Cleaned up this file.